image_rotate/include/image_rotate/image_rotate_nodelet.hpp

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
Nodelet is probably not the correct designation for this anymore since there is no longer a concept of a "nodelet" in ROS2. I would just say `_node`.

Reply:
There will have filename changes and not easy for review. So just update in a separate commit, fixed on https://github.com/ros-perception/image_pipeline/pull/385/commits/727347be3a393c05d3cdb6cb483b93565d244722